build_tools/dockerbuild.sh

```diff
@@ -1,2 +1,3 @@
 #!/usr/bin/env bash
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
Should these be at the very beginning of each file?

Reply:
Only needs to be in the first 7 lines, and for a shell script the shebang interpreter should come first.

Reply:
SGTM.